### DISCUSSION TITLE
Give an option to disable random SMTP from email address

### DISCUSSION THREAD

**Discussion by dreamerblue** [outline/outline#7971] [Ideas]
Related changes: #7844

Hi @tommoor, I recently upgraded to the latest 0.81.0 and then all my users are unable to receive emails. After troubleshooting, I see these reported errors:

```plain
Mail command failed: 436 \"MAIL FROM\" doesn't conform with authentication [@sm060104] (Auth Account:$SMTP_FROM_EMAIL|Mail Account:noreply-0juR9lgUnHRo7AjyYKN0mT8o@$MY_SMTP_FROM_DOMAIN)"
```

My email service provider doesn't allow arbitrary "from email address" to be specified (must apply and register in advance), so my request was denied.

Considering that some users don't care about security issues related to #7844 and they can't use random from email adress, I'm requesting to add an environment variable to disable this feature and use a fixed from email adress and reply email address.

**Comment by tommoor** [comment]
There is no need to create an option, this change has already been reverted on `main` for self-hosted instances and will be included in the next release 👍

**Reply by dreamerblue** [reply]
Thanks for the fixing, looking forward to the next release.
